02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,70 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Yvonne Ridenhour</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
 
-</body>
+  <header>
+    <h1>Yvonne Ridenhour | Consultant</h1>
+    <h5>1234 Taylor Drive, Austin, TX </h5>
+<hr>
+    <h3>CONSULTING STRENGTHS</h3>
+
+      <ul>
+
+        <li>Exceptional analytical and problem-solving business acumen; proven ability to form
+        recommendations to achieve outstanding results and neutralize obstacles across all industries.</li>
+
+        <li>Strong leader with ability to create highly motivated and effective operational teams.</li>
+
+        <li>Skill for staying in front of rapidly evolving operational requirements, quickly identifying needs, mplementing effective solutions and reengineering processes for improved business
+        performance.</li>
+
+      </ul>
+
+    <h3>EDUCATION</h3>
+
+      <ul>
+        <li>B.A. Liberal Arts</li>
+        <li>B.S. Education</li>
+
+      </ul>
+
+    <h3>FAVORITE QUOTE</h3>
+      <p>"People will forget what you said and what you did, but people will never forget how you made them feel."  Maya Angelou</p>
+<hr>
+
+    <h3>HOBBIES</h3>
+
+      <ul>
+
+        <li>Creating Small Planted Aquariums</li>
+        <li>Hiking</li>
+        <li>Photography</li>
+
+      </ul>
+
+      <h3>My Pets</h3>
+
+        <ul>
+
+          <li>Luki</li>
```

Comment:
Looking good, Yvonne! Some of the nesting is good, but there are some issues I'd like to see addressed to make sure you're on the right track. You've opened up a header tag on line 9 but never close it. Remember that only elements inside other elements get tabbed in. For example, in line 39, you have an h3 and then two more lines down, you've a ul that's tabbed in like it's inside the h3 when it is not. In the other direction, your images that are in the ul need to be inside an li tag structured like so:

<li>
  <img src..../>
</li>

or even

<li><img src=.../></li>

keep up the good work and make these adjustments please :)